sdk/resources/mgmt/src/main/java/com/azure/resourcemanager/AzureServiceClient.java

```diff
@@ -64,7 +64,7 @@ public String userAgent() {
     private static final String OS_NAME;
     private static final String OS_VERSION;
     private static final String JAVA_VERSION;
-    private static final String SDK_VERSION = "2.0.0-beta.2";
+    private static final String SDK_VERSION = "2.0.0-SNAPSHOT";
```

Comment:
@weidongxu-microsoft @xccc-msft @yungezz Do you think it need to change?

Reply:
vote for change to SNAPSHOT